cfme/configure/access_control.py

```diff
@@ -1261,7 +1259,9 @@ def is_displayed(self):
 class AllTenantView(ConfigurationView):
     """ All Tenants View """
     toolbar = View.nested(AccessControlToolbar)
-    table = Table('//*[@id="fieldset"]/table')
+    table = Table(VersionPick(
+        {Version.lowest(): '//*[@id="records_div"]/table',
```

Comment:
**REQUIRED** why the change from `//*[@id="fieldset"]/table` to `//*[@id="records_div"]/table`

Reply:
Its the updated xpath, there were no tests using this table, so the incorrect xpath was not noticed